### PR TITLE
*: add read_byte for ScanInfo

### PR DESCRIPTION
**Pull request by jyz0309**
Signed-off-by: jyz0309 <45495947@qq.com>

add read_byte for ScanInfo

### CODE REVIEW

**Comment by jyz0309** [comment]
@breeswish @crazycs520 PTAL

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -870,6 +870,7 @@ message TimeDetail {
 message ScanInfo {
     int64 total = 1;
     int64 processed = 2;
+    int64 read_byte =3;
```
Comment (breeswish) [inline proto/kvrpcpb.proto:873]:
```suggestion
    int64 read_bytes = 3;
```

Comment (jyz0309) [reply proto/kvrpcpb.proto]: done.

**Review by breeswish — APPROVED** [review]

**Review by crazycs520 — APPROVED** [review]
/lgtm

**Pull request merged**